test/jdk/java/lang/Thread/BuilderTest.java

```diff
@@ -24,8 +24,9 @@
 /**
  * @test
  * @summary Unit test for Thread.Builder
- * @compile --enable-preview -source ${jdk.version} BuilderTest.java
- * @run testng/othervm --enable-preview BuilderTest
+ * @enablePreview
+ * @compile BuilderTest.java
+ * @run testng/othervm BuilderTest
```

Comment:
`@enablePreview` wasn't available when most of these tests were initially created. I changed some of to use this tag as part of other changes but didn't cover over the remaining tests that still use `@compile --enable-preview -source ${jdk.version} ...` and `@run main/othervm --enable-preview ...`.   I just scanned the tests in test/jdk/java/lang/Thread/virtual and they can all be changed the same way, if you want. If you leave it then we'll just change them at the next edit in this area.

Reply:
I think I got most of those in the PR. There was a file or two I didn't update because not all the @compile/@run directives in the file used --enablePreview and the @enablePreview directive applies to all @compile and @run directives.

Reply:
Most of the tests in test/jdk/java/lang/Thread/virtual/ still have `@compile` and othervm, is it possible you didn't push the update with those changes?